geomesa-kafka/geomesa-kafka-datastore/geomesa-kafka-08-datastore/src/main/scala/org/locationtech/geomesa/kafka08/LiveKafkaConsumerFeatureSource.scala

```diff
@@ -38,10 +39,11 @@ class LiveKafkaConsumerFeatureSource(e: ContentEntry,
                                     (implicit ticker: Ticker = Ticker.systemTicker())
   extends KafkaConsumerFeatureSource(e, sft, q, monitor) with Runnable with Closeable with LazyLogging {
 
-  private[kafka08] val featureCache: LiveFeatureCache = if (useCQCache)
+  private [kafka08] val featureCache: LiveFeatureCache = if (useCQCache) {
```

Comment:
[picky] is there normally a space here? after private

Reply:
haha, i always put i space - looks weird to me without it. i'll revert